.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": ["plugin:jsx-a11y/recommended", "next/core-web-vitals"]
```

Comment:
What ESLINT rules are included in both of these?

Reply:
ESLINT Rules included:

- next plugin: https://nextjs.org/docs/pages/building-your-application/configuring/eslint#eslint-plugin
- eslint recommended rules identified by green checkmark: https://eslint.org/docs/latest/rules/
- Jsx-A11Y: https://www.npmjs.com/package/eslint-plugin-jsx-a11y#supported-rules